src/Button/Button.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import faker from "faker";
+import { render, fireEvent } from "@testing-library/react";
+import { Button } from "./Button";
+import { ContentProvider } from "../Content/Content";
+
+const TestButton = ({
+  contentNodes = {},
+  theme = {},
+  lang = "en",
+  contentKey,
+  ...rest
+}) => {
+  const defaultContentNodes = {};
+
+  const defaultTheme = {
+    colors: {
+      primary: {
+        light: "#d0dbfd",
+        dark: "#230cc2"
+      }
+    }
+  };
+
+  return (
+    <ContentProvider
+      content={{
+        theme: { ...defaultTheme, ...theme },
+        contentNodes: { ...defaultContentNodes, ...contentNodes }
+      }}
+      lang={lang}
+    >
+      <Button contentKey={contentKey} {...rest} />
+    </ContentProvider>
+  );
+};
```

Comment:
This setup would be on most tests. Would be nice to pull this out into somewhere that it can be a HOC. 
Like the test-utils here https://testing-library.com/docs/react-testing-library/setup#custom-render

Reply:
Updated! Thanks 👍